src/app/static/src/views/Home.vue

```diff
@@ -1,19 +1,28 @@
 <template>
   <div class="home">
-    <img alt="Vue logo" src="../assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js + TypeScript App"/>
+    <Charts v-if="metadata"
+            :chart-metadata="metadata.charts"
+            :chart-data="results"
+            :layout-data="chartLayoutData"></Charts>
   </div>
 </template>
 
 <script lang="ts">
 import { defineComponent } from "vue";
-import HelloWorld from "@/components/HelloWorld.vue";
```

Comment:
Probably time for a ticket to expunge this component and all the associated `increment`, `count` etc.

Reply:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-2372